cmd/go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/onsi/gomega v1.7.1 // indirect
 	github.com/pquerna/ffjson v0.0.0-20171002144729-d49c2bc1aa13 // indirect
 	github.com/prometheus/client_golang v1.8.0
-	github.com/stretchr/testify v1.6.1
+	github.com/stretchr/testify v1.7.0
```

Comment:
Do not update unrelated library, please.

Reply:
Sorry，I didn't notice that,I have fixed it.